client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/network/TransportFrameDecoderWithBufferSupplier.java

```diff
@@ -144,6 +148,9 @@ public void channelRead(ChannelHandlerContext ctx, Object data) {
           }
         }
       }
+    } catch (Exception e) {
+      logger.error(
+          "decode msg failed type {} curMsg {} size {}", curType, curMsg, nettyBuf.readableBytes());
```

Comment:
logger.error("", nettyBuf.readableBytes() => logger.error("", nettyBuf.readableBytes(),e)?, we can see the exception cause

Reply:
Yes,it should.